app/routes/app/project/issue/create.js

```diff
@@ -88,8 +88,9 @@ export default App.extend({
 
         let params = this.paramsFor('app.project.issue.edit');
 
+        let issue = this.setDefaultStatusToIssue(_self.statuses, _self.issue, 'new');
```

Comment:
Isn't it possible to set this in the model definition in https://github.com/projects4me/prometheus/blob/dev/app/models/issue.js?

Reply:
Changes are added!